breathe/renderer/sphinxrenderer.py

```diff
@@ -2048,15 +2048,22 @@ def visit_param(self, node: compound.paramTypeSub, *,
                         ''.join(n.astext() for n in nodelist),
                         location=self.state.state_machine.get_source_and_line(),
                         config=self.app.config)
-                    ast = parser._parse_type(named='single', outer='templateParam')
-                    assert ast.name is None
-                    nn = cpp.ASTNestedName(
-                        names=[cpp.ASTNestedNameElement(cpp.ASTIdentifier(node.declname), None)],
-                        templates=[False], rooted=False)
-                    ast.name = nn
-                    # the actual nodes don't matter, as it is astext()-ed later
-                    nodelist = [nodes.Text(str(ast))]
-                    appendDeclName = False
+                    try:
+                        # we really should use _parse_template_paramter()
```

Comment:
```suggestion
                        # we really should use _parse_template_parameter()
```

Reply:
Thanks

Reply:
this is not a typo from breathe's end.

https://github.com/sphinx-doc/sphinx/blob/7d7c59aaf22f110bfc7ed5a1385e6865ccf327fa/sphinx/domains/cpp.py#L6544